tests/test_debug.py

```diff
@@ -4,7 +4,7 @@
 from substratest.factory import Permissions
 
 
-@pytest.mark.skip("Need to fix this test")
+# @pytest.mark.skip("Need to fix this test")
```

Comment:
@Esadruhn This comment looks like a leftover now, no?

Reply:
Yes, we can remove it in the next PR ? Or I can do a small one to remove it now

Reply:
As you want !